src/server/common/model/section/section-model/section-model-v1.js

```diff
@@ -0,0 +1,142 @@
+import { QuestionPage } from '~/src/server/common/model/page/question-page-model.js'
+import { ExitPage } from '~/src/server/common/model/page/exit-page-model.js'
+import SummaryPage from '~/src/server/common/model/page/summary-page/SummaryPageModel.js'
+import { HiddenAnswer } from '../../answer/hidden/hidden.js'
+import { SectionModel } from './section-model.js'
+
+/**
+ * @import { Request } from '@hapi/hapi'
+ * @import { SectionValidation } from './section-model.js'
+ * @import { Page } from '~/src/server/common/model/page/page-model.js'
+ * @import {RawApplicationState} from '~/src/server/common/model/state/state-manager.js'
+ * @import { SectionData, QuestionPageAnswer, SectionPayload } from './section-model.js'
```

Comment:
`QuestionPageAnswer` should only be used local to this file, so would be better to move it here at some point.

It's not like a generic concept that is used outside of the section model anymore.